config/default/manager_auth_proxy_patch.yaml

```diff
@@ -16,7 +16,7 @@ spec:
             capabilities:
               drop:
                 - "ALL"
-          image: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.1
+          image: gcr.io/kubebuilder/kube-rbac-proxy:v0.14.1
```

Comment:
Updating to v0.14.1 as the docker image for v0.14.2 is [not found](https://console.cloud.google.com/gcr/images/kubebuilder/GLOBAL/kube-rbac-proxy:v0.14.1/details?tab=vulnz) on gcr.io